amazon-kinesis-client/src/test/java/software/amazon/kinesis/config/ReleaseCanaryPollingH1TestConfig.java

```diff
@@ -0,0 +1,37 @@
+package software.amazon.kinesis.config;
+
+import software.amazon.awssdk.http.Protocol;
+import software.amazon.kinesis.common.InitialPositionInStreamExtended;
+import software.amazon.kinesis.retrieval.RetrievalConfig;
+import software.amazon.kinesis.retrieval.polling.PollingConfig;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+/**
+ * Config for a polling consumer with HTTP protocol of HTTP1
+ */
+public class ReleaseCanaryPollingH1TestConfig extends KCLAppConfig {
```

Comment:
Are these names still unclear? Not sure what I could change them to to make it more clear.